config/settings/dev.py

```diff
@@ -90,3 +90,7 @@
 
 FORCE_PROCONNECT_LOGIN = os.getenv("FORCE_PROCONNECT_LOGIN", "True") == "True"
 REQUIRE_OTP_FOR_STAFF = os.getenv("REQUIRE_OTP_FOR_STAFF", "False") == "True"
+
+SHELL_PLUS_IMPORTS = [
+    "from datetime import datetime, timedelta",
```

Comment:
J'utilise pas spécialement donc +0 mais je pense qu'importer le module serait mieux, ça évitera de devoir modifier cette ligne dès qu'un symbole va manquer (_i.e._ `date` ou `datetime.UTC `) et les ambiguïtés entre le module et la classe
```suggestion
    "import datetime",
```